file-attribute-auditor/README.md

```diff
@@ -0,0 +1,15 @@
+# File Attribute Auditor
+
+The attribute audit notebook aims to provide insight into the following 3 areas:
+
+**Top level review of attribute key names and associated file counts:** this view can be utilized to determine how many files comply with business rules regarding application of file attributes. For example, do all ingested files have critical instrument details associated with them like Instrument Vendor and Instrument Model? 
+
+**Detailed breakdown of attribute key names, associated values and associated file counts:** this view can be utilized to identify fragmentation across file attribute labels. For example, how many files have ThermoFisher Scientific set as Instrument Vendor as opposed to Thermo Fisher Scientific?
+
+**Summary of File-Log Agent scanline file attribute assignments:** this view can be utilized to identify file attribute assignments at the data ingestion source so that new files that are uploaded to the platform with the desired attributes
+
+## How to Use
```

Comment:
Should the instructions say to clear the output before running to avoid confusion? (Though I like the idea of keeping the output for initial reference, however, even just viewing on Github.)